test-runner-poc/environments.js

```diff
@@ -140,10 +140,10 @@ class ShardedEnvironment extends EnvironmentBase {
   setup(callback) {
     const shardOptions = this.options && this.options.shard ? this.options.shard : {};
     // First set of nodes
-    const nodes1 = generateNodesArray(parsedURI.hosts, genShardedConfig, shardOptions);
+    //const nodes1 = generateNodesArray(parsedURI.hosts, genShardedConfig);
```

Comment:
Does everything work with this commented out?

Reply:
Commenting this line out doesn't affect the total number of errors when running a sharded instance.